reference/7.4/PSReadLine/Set-PSReadLineOption.md

```diff
@@ -748,6 +749,32 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -TerminateOrphanedConsoleApps
+
+This parameter sets the **TerminateOrphanedConsoleApps** option to `$true`.
+
+On Windows, when you press <kbd>Ctrl</kbd>+<kbd>c</kbd> to terminate a process, each process
+attached to a console receives a terminate signal, as opposed to just the active shell. Sometimes,
+when a shell has launched some large tree of child processes, (imagine a build system, for example)
+some processes may exit, leaving multiple processes concurrently trying to consume console input.
+
+When the **TerminateOrphanedConsoleApps** option is set to `$true`, tracks child processes and
+properly terminates them after receiving a <kbd>Ctrl</kbd>+<kbd>c</kbd> signal.
```

Comment:
```suggestion
When setting the **TerminateOrphanedConsoleApps** option to `$true`, PSReadLine records the list of processes that are currently attached to the console. Afterwards, whenever PSReadLine runs, it will get a new list of processes attached to the console, and terminate those that are not in the original list.
```